src/index.ts

```diff
@@ -89,3 +114,16 @@ export function geometryToCells(geometry, resolution: bigint): Quadbin[] {
     max_zoom: zoom
   }).map(([x, y, z]) => tileToCell({x, y, z}));
 }
+
+export function cellToBoundary(cell: bigint): Polygon {
```

Comment:
Could you add this (and the 3 new methods) to the `README`?

Reply:
Sure!